pkg/proxy/userspace/proxysocket.go

```diff
@@ -290,7 +290,7 @@ func (udp *udpProxySocket) proxyClient(cliAddr net.Addr, svrConn net.Conn, activ
 			klog.Errorf("SetDeadline failed: %v", err)
 			break
 		}
-		n, err = udp.WriteTo(buffer[0:n], cliAddr)
+		_, err = udp.WriteTo(buffer[0:n], cliAddr)
```

Comment:
Why are we ignoring this output? Is the linter complaining that it is not used?

Reply:
Yes, from the commit message:
```
pkg/proxy/userspace/proxysocket.go:293:3: this value of n is never used (SA4006)
```